chapters/en/slides/module5/module5_02.md

```diff
@@ -29,52 +34,78 @@ We will use the repository you just created as an example for the rest of this m
 ## Seeing how branches are related (terminal)
 
 <br>
-<br>
-<br>
+
+`git log --oneline --all --graph`
 
 <center>
 
-<img src='/module5/branches-related.png' width="600px" alt="404 image"/>
+<img src='/module5/branches-history-jl-t.png' width="800px" alt="404 image"/>
 
 </center>
 
 Notes:
 
 
-For this section we are going to simulate that we start working on a project. After cloning the repository for the first time we want to see what the branches look like. To see all branches as well as a graph indicating their relationship, we can type `git log --oneline --all --graph` in the terminal.
+For this section we are going to simulate that we start working on a project.
+After cloning the repository for  
+the first time we want to see what the branches look like. 
+To see all branches as well as a graph indicating their relationship,
+we can type `git log --oneline --all --graph` in the terminal.
 
-We can easily detect all the different branches (for example, `origin/hosting-revisions` means that the remote branch `hosting-revisions` it is in the commit 558114c)
+You can clone any repository on github.com 
+and run this command to visualize the project history. 
+For example, in the image we are visualizing 
+the repository of the [open source project Glosario](https://github.com/carpentries/glosario)  
+using the History tab of JupyterLab and   
+also the command `git log --oneline --all --graph` in the terminal.  
+Glosario is an open source glossary of terms used in data science  
+that [is available online](https://glosario.carpentries.org/)  
+ and also as a library in both R and Python.  
 
-Remember that you can use `space` on your keyboard to page down through the commit history and `b` to page up. Press `q` to exit back out to the terminal.
+Remember that you can use `space` on your keyboard
+to page down through the commit history and `b` to page up. 
+Press `q` to exit back out to the terminal.
 
 ---
 
 ## Seeing how branches are related (VS Code)
```

Comment:
It is actually possible to do this in JupyterLab now since version 0.39 of the git extension https://github.com/jupyterlab/jupyterlab-git/releases/tag/v0.39.0. I don't think we need to change anything now for the toolbox course (we can't keep up with all new changes right away), but maybe something that could be useful for 521 or get into this course in the future if we do updates?

Reply:
Yes, also the VSCode extension we use to visualize branches could be changed to GitLens, but I understand that to use it with private repos you should pay.